app/src/main/java/info/nightscout/androidaps/MainApp.kt

```diff
@@ -87,9 +87,9 @@ class MainApp : DaggerApplication() {
     }
 
     private fun selectThemeMode() {
-        when(sp.getString(R.string.key_use_darkmode, "system")) {
-            "dark" -> AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES);
-            "light" -> AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO);
+        when(sp.getString(R.string.key_useDarkmode, "system")) {
+            sp.getString(R.string.key_Dark, "dark") -> AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES);
```

Comment:
So you've changed this from a string which should not be translated, to a getString (which is used for translation) but marked it `translatable="false"`, and provided a default value, which is the same as the key value?

Note that this is essentially a switch-case statement, and we are checking the value of `R.string.key_useDarkmode`, with a default of `"system"`, based on the values in the [array here](https://github.com/nightscout/AndroidAPS/pull/326/files/83487fa474ec4566bdfa7239aefa7d6ca9e2795a..fba201c0e2fad60146f146f5e6bb8b04497863fb#diff-811c0a2f240d097b6a9c64caf939bacde3c2361f41f8ecb3df2d35af83f67f01R150-R154).

Reply:
can you make a PR to megre yoir new changes into my branch ?